src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -31,19 +32,35 @@ public void init() {
   }
 
   @Override
-  // Get a specific user. Return null if not found. TODO: User Validation
+  // Get a specific user. Return null if not found.
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGithub = request.getParameter("githubID");
+    String userGithub = request.getParameter("githubID"); // TODO:Check for null values
     Key userKey = KeyFactory.createKey(PersonBuilder.ENTITY_NAME, userGithub);
-    System.out.println(userKey);
 
+    Entity userEntity; // TODO: Abstract doGet() into it's own function
+    try {
+      userEntity = datastore.get(userKey);
+    } catch (Exception e) {
```

Comment:
Is it possible to be a little more specific about what Exceptions we expect to be thrown here? For example, it should just be EntityNotFoundException here. https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/EntityNotFoundException.html

And below in the other try/catches.